packages/fxa-content-server/app/styles/modules/_graphic.scss

```diff
@@ -152,3 +152,13 @@
   height: 24px;
   margin-left: 3px;
 }
+
+#appleid-signin {
+  width: 210px;
+  height: 48px;
+}
+
+#appleid-signin > div > div > svg {
+  height: 50px;
+  width: 100%;
+}
```

Comment:
This will complain at you for using an ID to style with, but I don't really mind since we're going to get rid of all of this hopefully this year 🤞

You could at least do:
```
#appleid-signin {
  width: 210px;
  height: 48px;

  svg {
    height: 50px;
    width: 100%;
  }
}
```
Maybe you can at least do something like this?